modules/understanding-vsphere-problem-detector.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_vsphere/using-vsphere-problem-detector-operator.adoc
+
+:operator-name: vSphere Problem Detector Operator
+
+[id="understanding-vsphere-problem-detector_{context}"]
+= Understanding the vSphere Problem Detector Operator
+
+The {operator-name} checks clusters that are deployed on vSphere for common installation and misconfiguration issues that are related to storage.
+
+The Operator runs in the `openshift-cluster-storage-operator` namespace and is started by the `cluster-storage-operator` when the Cluster Storage Operator detects that the cluster is deployed on vSphere. The {operator-name} communicates with the vSphere vCenter Server to determine the virtual machines in the cluster, the default datastore, and other information about the vSphere vCenter Server configuration. The Operator uses the credentials from the `cloud-credentials-provider` to connect to vSphere.
+
```

Comment:
credentials from the `cloud credentials operator`?

Reply:
@jsafrane  is Ping correct or is it "vsphere-cloud-credentials"?  My interest is to let customers know which credentials to check if the Operator cannot communicate with vSphere.

Reply:
@gnufied , Bob told me that Jan might not be around for a bit.  Is this something you can clarify?

Can you share how to get the correct answer so that I can be more effective in the future?  Thank you.

Reply:
It's cloud-credential-operator, https://github.com/openshift/cloud-credential-operator